test/unittest/layers/layers_standalone_common_tests.cpp

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2020 Jihoon Lee <jhoon.it.lee@samsung.com>
+ *
+ * @file layer_common_tests.cpp
+ * @date 15 June 2021
+ * @brief Common test for nntrainer layers (Param Tests)
+ * @see	https://github.com/nnstreamer/nntrainer
+ * @author Jihoon Lee <jhoon.it.lee@samsung.com>
+ * @bug No known bugs except for NYI items
+ */
+
+#include <layers_common_tests.h>
+
+#include <app_context.h>
+#include <layer_devel.h>
+
+constexpr unsigned SAMPLE_TRIES = 10;
+
+TEST_P(LayerSemantics, createFromAppContext_pn) {
+  auto &ac = nntrainer::AppContext::Global();
```

Comment:
I guess App context is dependent test?

Reply:
moved.

Reply:
also moved layer context from LayerSemantics to dependent unittest.
Now LayerSemantics and standalone_unittest is not dependent on app_context.

Reply:
Cool